compiler/src/test-integration/java/io/neow3j/compiler/VerificationMethodIntegrationTest.java

```diff
@@ -21,14 +22,25 @@ public static void setUp() throws Throwable {
         setUp(VerificationMethodIntegrationTestContract.class.getName());
         // The RPC method invokecontractverify requires an open wallet on the
         // neo-node.
-        neow3j.openWallet("wallet.json","neo").send();
+        neow3j.openWallet("wallet.json", "neo").send();
     }
 
     @Test
     public void callVerifyWithContractOwner() throws Throwable {
         NeoInvokeContractVerify response = neow3j
                 .invokeContractVerify(contract.getScriptHash().toString(),
-                        Arrays.asList(string("hello, world!")),
```

Comment:
With this pr, the api is extended, such that many methods now also accept `Hash160` or `Hash256` instead of just `String`.
One test covers `invokeContractVerify(String, ...)` and one covers `invokeContractVerify(Hash160, ...)`. Ultimately, the one taking a String will call the other one, but I think it still makes sense to have one test for all methods separately.

Reply:
👍 
In that cas the test `callVerifyWithContractOwner()` should not do 
```
.invokeContractVerify(contract.getScriptHash().toString(),
```
but
```
.invokeContractVerify(contract.getScriptHash(),
```
right?

Reply:
The two tests `callVerifyWithOtherSigner()` and `callVerifyWithOtherSigner_fromString()` still contain the exact same code.